pyproject.toml

```diff
@@ -57,13 +57,16 @@ pytest-cov = {version = ">=5.0.0", optional = true}
 datasets = ">=2.19.0"
 imagecodecs = { version = ">=2024.1.1", optional = true }
 pyav = ">=12.0.5"
-moviepy = ">=1.0.3"
 rerun-sdk = ">=0.15.1"
 deepdiff = ">=7.0.1"
-scikit-image = {version = ">=0.23.2", optional = true}
+flask = ">=3.0.3"
 pandas = {version = ">=2.2.2", optional = true}
+pytest-mock = {version = ">=3.14.0", optional = true}
```

Comment:
Do you use this? If not, I think it's a merge issue as I removed it recently.
Also, backward compatibility tests are not passing currently. You probably need to re-generate the artifacts.